gwpy/cli/qtransform.py

```diff
@@ -112,12 +112,24 @@ def _finalize_arguments(self, args):
         """
         gps = args.gps
         search = args.search
-        # ennsure we have enough data for filter settling
+        # ensure we have enough data for filter settling
         max_plot = max(args.plot)
         search = max(search, max_plot * 2 + 6)
+        args.search = search
         self.log(3, "Search window: {0:.0f} sec, max plot window {1:.0f}".
                  format(search, max_plot))
 
+        # make sure we don't create too big interpolations
+        try:
+            m = re.match('(\\d+)x(\\d+)', args.geometry)
```

Comment:
@areeda, please move this statement outside of the `try/except` block, since we don't want to catch `AttributeError` here.

Reply:
This is now back to if statements which will triger a complexity notice but I think it's easier to understand